src/components/taskDetails/index.tsx

```diff
@@ -166,22 +160,10 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
 
     const shouldRenderParentContainer = () => !isLoading && !isError && data;
 
-    const { data: taskProgress } = useGetProgressDetailsQuery({
+    const { data: progressDate } = useGetProgressDetailsQuery({
```

Comment:
I think it's "progressData" not "progressDate". Correct me if I am wrong.

Reply:
Yes you are correct. I have changed it.